modules/standard/IO.chpl

```diff
@@ -2847,14 +2847,8 @@ proc _channel.withFormatter(f: ?) {
   return ret;
 }
 
-/*
-
-Represents a Unicode codepoint. I/O routines (such as :proc:`channel.read`
```

Comment:
IMO it's worth leaving a comment explaining why this exists, for future developers.